.github/workflows/npm-alpha.yml

```diff
@@ -10,25 +10,34 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: 12.x
-          registry-url: https://npm.pkg.github.com/
+          always-auth: true
+      - run: yarn install
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.CD_NPM_TOKEN }}
```

Comment:
for install, do we need auth? especially now that the github registry reference are all gone from `yarn.lock` @vxsl

Reply:
good point will revise

Reply:
each workflow file has been revised to ensure deps are installing from default registry without auth :+1: